overlord/snapstate/snapmgr.go

```diff
@@ -433,6 +433,7 @@ func Manager(st *state.State, runner *state.TaskRunner) (*SnapManager, error) {
 	runner.AddCleanup("copy-snap-data", m.cleanupCopySnapData)
 	runner.AddHandler("link-snap", m.doLinkSnap, m.undoLinkSnap)
 	runner.AddHandler("start-snap-services", m.startSnapServices, m.stopSnapServices)
+	runner.AddHandler("service-control", m.doServiceControl, nil)
```

Comment:
do we need an undo?

Reply:
I think we do, although I wanted to attack it separately. I suppose it will be needed for multi-snap ops executed by `snap` cli (such as `snap stop snap1.service1 snap2.service2), but I'm not sure if it would used for service commands from hooks.